build.sbt

```diff
@@ -989,6 +989,28 @@ lazy val `interpreter-dsl` = (project in file("lib/scala/interpreter-dsl"))
     )
   )
 
+lazy val `interpreter-dsl-test` =
+  (project in file("engine/interpreter-dsl-test"))
+    .configs(Test)
+    .settings(
+      version := "0.1",
+      frgaalJavaCompilerSetting,
+      Test / fork := true,
+      Test / javaOptions ++= Seq(
+        "-Dgraalvm.locatorDisabled=true",
+        s"--upgrade-module-path=${file("engine/runtime/build-cache/truffle-api.jar").absolutePath}"
+      ),
+      commands += WithDebugCommand.withDebug,
+      libraryDependencies ++= Seq(
+        "org.graalvm.truffle" % "truffle-api"           % graalVersion % "provided",
+        "org.graalvm.truffle" % "truffle-dsl-processor" % graalVersion % "provided",
+        "junit"               % "junit"                 % junitVersion % Test,
+        "com.novocode"        % "junit-interface"       % "0.11"       % Test exclude ("junit", "junit-dep")
```

Comment:
The `junit-interface` dependency is already used in two projects. Please refactor the version into the `junitInterfaceVersion` variable. This way it is much easier when it comes to upgrading the dependencies.

Reply:
Done in [455d12e](https://github.com/enso-org/enso/pull/6304/commits/455d12e4b178d439eaaaecf22735f2ee5c43009d)